admin/sql/create_models.sql

```diff
@@ -19,4 +19,96 @@ INSERT INTO model (model, model_version, date, status) VALUES ('timbre', 'v2.1_b
 INSERT INTO model (model, model_version, date, status) VALUES ('tonal_atonal', 'v2.1_beta1', now(), 'show');
 INSERT INTO model (model, model_version, date, status) VALUES ('voice_instrumental', 'v2.1_beta1', now(), 'show');
 
+UPDATE model SET class_mapping = '{
+        "danceable": "Danceable",
+        "not_danceable": "Not danceable"
+    }'::jsonb WHERE model.model = 'danceability';
+
+UPDATE model SET class_mapping = '{
+        "acoustic": "Acoustic",
+        "not_acoustic": "Not acoustic"
```

Comment:
I also updated these items so that we don't have to automatically remove _ from them.